src/types/RecordType.js.flow

```diff
@@ -4,6 +4,7 @@ import Type from './Type'
 
 declare class RecordType<K: string | number | symbol, V> extends Type<{
   [K]: V,
+  ...
```

Comment:
Good call on all of this!  The only issue is I actually should have made this an exact object type.  I'll change it and merge, just wanted to give you a heads up